library/src/main/java/com/chuckerteam/chucker/api/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -129,19 +129,45 @@ internal class TransactionPayloadFragment : Fragment(), TransactionFragment, Sea
         return true
     }
 
-    private fun updateImageDataView(imageData: ByteArray?) {
-        val bitmap = imageData?.let {
-            BitmapFactory.decodeByteArray(imageData, 0, imageData.size)
+    private class UiLoaderTask(val fragment: TransactionPayloadFragment) :
+        AsyncTask<Pair<Int, HttpTransaction>, Unit, UiPayload>() {
+
+        override fun doInBackground(vararg params: Pair<Int, HttpTransaction>):
+        UiPayload {
+            val (type, transaction) = params[0]
+
+            val bitmap = transaction.responseImageData?.let { imageData ->
+                BitmapFactory.decodeByteArray(imageData, 0, imageData.size)
+            }
+
+            return when (type) {
+                TYPE_REQUEST -> UiPayload(
+                    transaction.getRequestHeadersString(true),
+                    transaction.getFormattedRequestBody(),
+                    transaction.isRequestBodyPlainText, bitmap
```

Comment:
Can you put it on 2 lines?
```suggestion
                    transaction.isRequestBodyPlainText, 
                    bitmap
```

Reply:
Damn. Sorry, our work project's setup make that happen automatically when formatting, I though the same would happen here :-P